extensions/amp-experiment/1.0/mutation/attribute-mutation-default-style.js

```diff
@@ -16,16 +16,36 @@
 
 import {assertAttributeMutationFormat} from './mutation';
 import {assertDoesNotContainDisplay, setStyles} from '../../../../src/style';
-import {dev} from '../../../../src/log';
+import {dev, user} from '../../../../src/log';
 import {dict, hasOwn} from '../../../../src/utils/object';
+import {isAmpElement} from '../../../../src/dom';
 
+/** @const {Object<string, RegExp>} */
 const SUPPORTED_STYLE_VALUE = {
-  'color': /#([A-Fa-f0-9]{6}|[A-Fa-f0-9]{3});?$/,
+  'color': /.*/,
```

Comment:
might worth to extract a regex constant for this to save CPU on regex parsing and instantiation.
or maybe even set it to `true` and skip the regex matching.

Reply:
Sounds good.

Reply:
Done. I wanted to set the value to `true` and skip the regex matching. But there's tradeoff there, the code looks less cleaner. Ended up extracting the regex constant.